setup.py

```diff
@@ -50,7 +50,7 @@
     },
     install_requires=[
         "cryptography",
-        "enum34",
+        "enum-compat",
```

Comment:
https://github.com/jstasiak/enum-compat/blob/master/README.rst suggests to use
enum34; python_version < '3.4'
directly.
With enum-compat, we get autogenerated RPM dep in Fedora Requires: python3.9dist(enum-compat) which does not exist.

Reply:
Ah, looks like it should be changed back to `enum34; python_version < '3.4'`. Looks like the README in enum-compat was updated a couple months after this was merged to say it's no longer needed.